specs/import_cluster_workflow_ui.adoc

```diff
@@ -0,0 +1,147 @@
+= Import cluster workflow by using tendrl-frontend UI
+
+The intent of import cluster workflow is to provide a mechanism to add an existing storage cluster(gluster, ceph etc) into tendrl for management using tendrl-frontend UI.
+
+
+== Problem description
+
+This specification describes import cluster workflow using UI which is a mechanism to add an existing storage cluster(gluster, ceph etc) into tendrl for management. This specification addresses the import use-cases of both ceph and gluster storage.
+
+
+== Use Cases
+
+This addresses the use case of importing existing clusters(ceph. gluster etc.) into tendrl for management.
+
+The workflow is as follows:
+
+* "Import Cluster" can be triggers from Landing page (First Time Experience) or Cluster List View.
+
+* User click on "Import Cluster". 
+
+* If multiple storage service providers have been installed on the system. Than it will ask for select the storage service(gluster/ceph) to use for cluster.
+
+* Now we have two step to go for import cluster
+
+	** Step 1: Now we will have all node listed. User select 1 host.
+	( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/183602753 )
+
+		*** Select any MON node in the cluster in case of ceph.
+		*** Select any node in the cluster in case of gluster.
+
+	** Step 2: Use the selected node (in previous step) to extract the details of the cluster.
+	( https://redhat.invisionapp.com/share/R88EUSGJK#/screens/193145495 )
+		*** Now we will have cluster summary. Now go for "Import Cluster".
+
+
+* If cluster associated with selected host contains unsupported configuration, System notifies user to select another cluster to import or to cancel import. 
```

Comment:
We need to capture the error scenarios as well. Please explain in details on case by case basis about the use cases you are trying address here, including the error cases.